src/System.Runtime.Analyzers/VisualBasic/System.Runtime.VisualBasic.Analyzers.vbproj

```diff
@@ -116,8 +116,8 @@
     <Compile Include="BasicRethrowToPreserveStackDetails.Fixer.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.Fixer.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.Fixer.vb" />
+    <Compile Include="BasicInitializeStaticFieldsInline.vb" />
```

Comment:
I thought this comment suggested having C# and VB files for corresponding analyzer implementations:

```
Concrete derived class CSharpFoo in CSharpFoo.cs (or CA9999_CSharpFoo.cs as proposed above)
Concrete derived class BasicFoo in BasicFoo.vb (or CA9999_BasicFoo.vb)
```